.gitignore

```diff
@@ -6,3 +6,5 @@ _books/*
 node_modules/*
 .idea/*
 .platform/local/*
+src/registry/images/examples/*
```

Comment:
@Crell They will so long as these two lines are removed. I included them as a sort of `platform build` failsafe and worked around this with the move commands. I don't care what we do here, but I still will occasionally forget and commit user-widget on PRs. I figured, .gitignore these, otherwise that situation is user-widget + 120 example files.